types/react-transition-group/Transition.d.ts

```diff
@@ -50,7 +139,7 @@ export interface TransitionProps extends TransitionActions {
  * It's up to you to give meaning and effect to those states. For example we can
  * add styles to a component when it enters or exits:
  *
- * ```jsx
+ * ```js
```

Comment:
I think `jsx` is more appropriate than `js` here.

Reply:
I found it strange when I was exploring how JSDoc is actually rendered in IDEs and in the case of VSCode it doesn't actually like JSX code fences (it doesn't recognize it). It will do syntax highlighting in the helper tooltip if I leave it as JS, however (it highlights JSX syntax correctly as well). I can turn it back though if we want to be super-general and not be VSCode specific; I didn't test it with other IDEs.

Reply:
I'm trying VSCode 1.33.1 on my MacBook, but it seems `jsx` renders syntax highlight correctly... 🤔 

Here is the screenshot:

![image](https://user-images.githubusercontent.com/473530/56845408-bab4c580-68fb-11e9-9da9-044a1c601196.png)

Reply:
Yep. They both highlight the same in the actual JSDoc markup, but the highlighting does not work for JSX in the actual tooltip:

The below is using `js`
![image](https://user-images.githubusercontent.com/12239873/56864919-c51f9f80-698d-11e9-8fdc-f712f9991b5b.png)

The below is using `jsx`
![image](https://user-images.githubusercontent.com/12239873/56864932-d8cb0600-698d-11e9-982d-6f2f6c01fceb.png)

Reply:
Hmm, it looks to me that this is a VSCode's bug... 🤔 
I think there are the following reasons that `jsx` is better than `js`:

- If this tooltip behavior is a VSCode's bug, it may be fixed in the future.
- It may not work well other IDEs or JSDoc (markdown) parser.
- Developers may be confusing because the original source code uses `jsx`. For example [here](https://github.com/reactjs/react-transition-group/blob/master/src/Transition.js#L36).